alf/examples/ppo_rnd_mrevenge_conf.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import alf
+import torch
+
+import functools
+
+# from alf.algorithms.actor_critic_algorithm import ActorCriticAlgorithm
+from alf.algorithms.rnd_algorithm import RNDAlgorithm
+from alf.networks import EncodingNetwork, ActorDistributionNetwork, CategoricalProjectionNetwork, ValueNetwork
+from alf.tensor_specs import TensorSpec
+
+from alf.examples import ppo_conf
```

Comment:
Also due to my previous PR, ppo_conf by default uses EntropyTargetAlgorithm. I think we'd better change it back to not use it, by

``` python
alf.config('Agent', rl_algorithm_cls=PPOAlgorithm, 
                 #enforce_entropy_target=True
)

#alf.config('EntropyTargetAlgorithm', initial_alpha=1.)
```